beamer/deploy/commands.py

```diff
@@ -182,7 +197,7 @@ def deploy(
     assert base_w3.eth.chain_id == base_deployment.base.chain_id
     log.info("Connected to base chain RPC", chain_id=base_deployment.base.chain_id, url=url)
 
-    resolver = base_deployment.obtain_contract(base_w3, "base", "Resolver")
+    resolver = obtain_contract(base_w3, abi_dir, base_deployment, "Resolver")
```

Comment:
question: Is the base deployment also stored as a `Deployment` object with `chain = None` ?

Reply:
> question: Is the base deployment also stored as a `Deployment` object with `chain = None` ?

Yes, that's exactly what differentiates the base deployment from non-base deployments.